### ISSUE TITLE
 ImportError: No module named 'sentry_airflow' and Failed to import plugin /opt/airflow/plugins/sentry-airflow/tests/test_sentry_hook.py

### ISSUE THREAD

**Issue by andilabs** [getsentry/sentry_airflow#20]
first of all is this plugin aimed to also work too with not self-hosted, but SaaS `sentry.io`?
I followed the instruction for seutp:
In my airflow docker container, I have configured 
`SENTRY_DSN="https://{KEY}@sentry.io/{PROJECT_ID}"`
I use:
```
apache-airflow==1.10.3
sentry-sdk==0.11.2
Flask==1.0.4
```
I got the following error:
```

# [2019-09-19 15:33:42,476] {plugins_manager.py:143} ERROR - No module named 'sentry_airflow'
# Traceback (most recent call last):
#   File "/z/datalake/virtual_envs/20190812T103918/Ubuntu16.04/lib/python3.5/site-packages/airflow/plugins_manager.py", line 137, in <module>
#     m = imp.load_source(namespace, filepath)
#   File "/z/datalake/virtual_envs/20190812T103918/Ubuntu16.04/lib/python3.5/imp.py", line 172, in load_source
#     module = _load(spec)
#   File "<frozen importlib._bootstrap>", line 693, in _load
#   File "<frozen importlib._bootstrap>", line 673, in _load_unlocked
#   File "<frozen importlib._bootstrap_external>", line 697, in exec_module
#   File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
#   File "/opt/airflow/plugins/sentry-airflow/__init__.py", line 2, in <module>
#     from sentry_airflow.hooks.sentry_hook import SentryHook
# ImportError: No module named 'sentry_airflow'
# [2019-09-19 15:33:42,657] {plugins_manager.py:144} ERROR - Failed to import plugin /opt/airflow/plugins/sentry-airflow/__init__.py
# [2019-09-19 15:33:42,852] {plugins_manager.py:143} ERROR - Unsupported scheme ''
# Traceback (most recent call last):
#   File "/opt/airflow/plugins/sentry-airflow/hooks/sentry_hook.py", line 118, in __init__
#     conn = self.get_connection(sentry_conn_id)
#   File "/z/datalake/virtual_envs/20190812T103918/Ubuntu16.04/lib/python3.5/site-packages/airflow/hooks/base_hook.py", line 80, in get_connection
#     conn = random.choice(cls.get_connections(conn_id))
#   File "/z/datalake/virtual_envs/20190812T103918/Ubuntu16.04/lib/python3.5/site-packages/airflow/hooks/base_hook.py", line 75, in get_connections
#     conns = cls._get_connections_from_db(conn_id)
#   File "/z/datalake/virtual_envs/20190812T103918/Ubuntu16.04/lib/python3.5/site-packages/airflow/utils/db.py", line 73, in wrapper
#     return func(*args, **kwargs)
#   File "/z/datalake/virtual_envs/20190812T103918/Ubuntu16.04/lib/python3.5/site-packages/airflow/hooks/base_hook.py", line 58, in _get_connections_from_db
#     "The conn_id `{0}` isn't defined".format(conn_id))
# airflow.exceptions.AirflowException: The conn_id `sentry_dsn` isn't defined
```

just for reference, when following this (https://pedrocarmona.github.io/blog/sentry-aiflow/) hacky-setup without plugin I was able to get the errors in my sentry.io instance

**Comment by andilabs** [comment]
so regarding the part with `'sentry_dsn` my fault, It should be ldeinfed like:
`SENTRY_DSN=xyz`

and not like:

`SENTRY_DSN="xyz"
`

**Now I got all the errors inside sentry.io, but also I get the following errors:**

- **`No module named 'sentry_airflow'`**:
```
ImportError: No module named 'sentry_airflow'
  File "airflow/plugins_manager.py", line 137, in <module>
    m = imp.load_source(namespace, filepath)
  File "imp.py", line 172, in load_source
    module = _load(spec)
  File "<frozen importlib._bootstrap>", line 693, in _load
  File "<frozen importlib._bootstrap>", line 673, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 697, in exec_module
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "test_sentry_hook.py", line 16, in <module>
    from sentry_airflow.hooks.sentry_hook import (
```
- **`Failed to import plugin /opt/airflow/plugins/sentry-airflow/tests/test_sentry_hook.py`**:

**Comment by andilabs** [comment]
I think the mater is about renaming the pulled plugin. I proposed PR https://github.com/getsentry/sentry_airflow/pull/21

**Comment by AbhiPrasad** [comment]
Fixed with #21 and #22
